<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatePlease</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>158</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="chatBox">
      <property name="title">
       <string>Chat for clients</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="4">
        <widget class="QLabel" name="memberLabel">
         <property name="text">
          <string>Members:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="2" colspan="3">
        <widget class="QTextBrowser" name="chatBrowser">
         <property name="minimumSize">
          <size>
           <width>363</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="sendChatButton">
         <property name="text">
          <string>&amp;send</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QTextBrowser" name="memberBrowser">
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="messageLabel">
         <property name="text">
          <string>Your &amp;message:</string>
         </property>
         <property name="buddy">
          <cstring>chatEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="chatEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>158</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="openChatButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>18</height>
       </size>
      </property>
      <property name="text">
       <string>Start chat with other clients</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="hideChatButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>18</height>
       </size>
      </property>
      <property name="text">
       <string>Hide chat</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>158</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>158</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="mainLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="askLabel">
            <property name="text">
             <string>&amp;Write just here:</string>
            </property>
            <property name="buddy">
             <cstring>askLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="askLineEdit">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;calculate</string>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Answer:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="answerText">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>170</width>
              <height>27</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>128</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="ipLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>server ip-&amp;address:</string>
          </property>
          <property name="buddy">
           <cstring>addressLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addressLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;connect</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnectButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;disconnect</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reportButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>send report</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Information</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAuthor"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionDisplay"/>
    <addaction name="actionDescription"/>
   </widget>
   <addaction name="menuMap"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDisplay">
   <property name="text">
    <string>Display</string>
   </property>
  </action>
  <action name="actionDescription">
   <property name="text">
    <string>Description</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
